Main Model/household-procedures.nls

```diff
@@ -6,7 +6,7 @@ to search-cheaper-vendor
   let current-trading-link one-of my-consumer-links
```

Comment:
since they are searching for a cheaper vendor, I think they should use `max-one-of consumer-link-neighbors [price]`